extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputRowParser.java

```diff
@@ -48,19 +49,29 @@
   private static final Logger LOG = new Logger(OpenCensusProtobufInputRowParser.class);
 
   private static final String SEPARATOR = "-";
-  public static final String NAME = "name";
-  public static final String VALUE = "value";
-  public static final String TIMESTAMP_COLUMN = "timestamp";
+  private static final String NAME = "name";
+  private static final String VALUE = "value";
+  private static final String TIMESTAMP_COLUMN = "timestamp";
+  private static final String DEFAULT_RESOURCE_PREFIX = "resource.";
   private final ParseSpec parseSpec;
   private final List<String> dimensions;
 
-  @JsonCreator
+  private final String metricDimension;
+  private final String descriptorDimensionPrefix;
+  private final String resourceDimensionPrefix;
+
   public OpenCensusProtobufInputRowParser(
-      @JsonProperty("parseSpec") ParseSpec parseSpec
+      @JsonProperty("parseSpec") ParseSpec parseSpec,
+      @JsonProperty("metricDimension") String metricDimension,
+      @JsonProperty("descriptorDimensionPrefix") String descriptorPrefix,
```

Comment:
I'm not sure if `descriptor` makes it very obvious this is referring to metric labels. I was wrong here, `metricLabelPrefix` and `resourceLabelPrefix` will probably be easier to understand.

Reply:
Done.

Reply:
I don't see this change

Reply:
Missed the commit. Done.